backend/substrapp/tasks/tasks_compute_task.py

```diff
@@ -195,27 +196,32 @@ def compute_task(self: ComputeTask, channel_name: str, serialized_task: str, com
         raise compute_task_errors.CeleryRetryError() from exception
 
 
-@retry()
-def _create_task_profiling(channel_name: str, compute_task_key: str) -> bytes:
-    parameters = {"compute_task_key": compute_task_key}
+def _send_profiling_event(*, channel_name: str, url_create: str, url_update: str, data: dict[str, Any]) -> bytes:
```

Comment:
is this shared, or just an event internal to the backend?

Reply:
Profiling events are only registered in the api of the same backend.